framework/core/less/forum/DiscussionPage.less

```diff
@@ -51,11 +59,9 @@
       }
     }
   }
-}
 
-@media @tablet-up {
   .DiscussionPage-stream {
-    margin-right: 225px;
+    order: -1;
```

Comment:
I'd prefer to use explicit grid columns or even grid areas here, to be more explicit in what is desired.